backend/CircleForms/Controllers/V1/PostsController.cs

```diff
@@ -148,6 +150,10 @@ public async Task<IActionResult> Post(PostRequestContract postContract)
             _logger.LogInformation("User {User} posts a post {PostId}", claim, post.ID);
 
             post.AuthorId = claim;
+            if (post.Accessibility == Accessibility.Link)
+            {
+                post.AccessKey = Guid.NewGuid().ToString();
```

Comment:
Might wanna trim it a bit? Not sure we want 16 digit key for this, something similar to soundcloud's 6 letters would be better user-wise. We already have long post IDs, don't think making links even longer is a good idea since people will have to use link shorteners.